src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1038,6 +1034,16 @@ private static Autogenerated.StateOptions.Types.StateConcurrency GetStateConcurr
                 _ => throw new ArgumentException($"{concurrencyMode} Concurrency Mode is not supported.")
             };
         }
+
+        private void AddDaprApiTokenHeader(Metadata headers)
+        {
+            var daprApiToken = Environment.GetEnvironmentVariable(Constants.DaprApiTokenEnvironmentVariable);
```

Comment:
Similar to a comment elsewhere, the source of truth should be a field on this class, not the env-var. The reason for providing an API for this is that it adds flexibility to do other things.

Example: Some people **really really** don't like security-related things in environment variables, they have an organization policy to ready them from files on an in memory file-system.